src/App.test.js

```diff
@@ -715,4 +716,50 @@ describe('App component', () => {
       expect(localStorageItems['dopplerFirstOrigin.value']).toBeUndefined();
     });
   });
+
+  describe('activation in progress parameter', () => {
```

Comment:
I think that it is not the right subject, try to replace the it with this value:

* activation in progress parameter should call google adwords
* activation in progress parameter  should not to be call google adwords

I think that _activation in progress parameter_ is not who calls google adwords